CHANGELOG/1.2-CHANGELOG.md

```diff
@@ -0,0 +1,431 @@
+* Add API for GEP-3171 by @jakebennert in [#3283](https://github.com/kubernetes-sigs/gateway-api/pull/3283)
+* Add Additional API Testing for GEP-3171 by @jakebennert in [#3302](https://github.com/kubernetes-sigs/gateway-api/pull/3302)
+
+### Improvements to backend TLS configuration
+
+Some changes have been made to Gateway API's support for
+configuring TLS connections between the Gateway and backends.
+
+Gateway has a new Experimental field that contains configuration
+for the client certificate Gateways should use when connecting
+to Backends.
+
+The existing BackendTLSPolicy object has had additions as well:
+
+* Ability to specify SANs on BackendTLSPolicy
+* Add TLS options to BackendTLSPolicy to mirror TLS config on Gateways
+
+See [GEP-3155](https://gateway-api.sigs.k8s.io/geps/gep-3155/) for all the details.
+
+This feature has graduated to Experimental amd should now be used for
+testing and verification purposes only. Experimental features may be
+changed or removed in a future version.
+
+This feature does not currently have a feature name defined.
+
+This feature's status is **Extended**, meaning that it is _optional_ for
+implementations to support.
+
+As there is no feature name or conformance tests available for this feature
+as yet, please see your implementation's documentation to see if it is
+supported.
+
+Relevant PRs:
+
+* Add GEP-3155: Complete Backend mTLS Configuration by @mkosieradzki in [#3180](https://github.com/kubernetes-sigs/gateway-api/pull/3180)
+* add api changes according to gep-3155 by @LiorLieberman in [#3304](https://github.com/kubernetes-sigs/gateway-api/pull/3304)
+
+### Named Route Rules
+
+All Route Rule types (GRPCRouteRule, HTTPRouteRule, TCPRouteRule, TLSRouteRule
+and UDPRouteRule) have had a new, optional `name` field to support referencing
+individual rules by name.
+
+This name, if present, may be used in `status` and logging to indicate which route
+rule is being referenced in messages in a more readable way than an array index.
+
+This feature has graduated to Experimental amd should now be used for
+testing and verification purposes only. Experimental features may be
+changed or removed in a future version.
+
+This feature does not currently have a feature name defined.
+
+This feature's status is **Extended**, meaning that it is _optional_ for
+implementations to support.
+
+As there is no feature name or conformance tests available for this feature
+as yet, please see your implementation's documentation to see if it is
+supported.
+
+Relevant PRs:
+
+* Implement of GEP-995 by @howardjohn in [#2985](https://github.com/kubernetes-sigs/gateway-api/pull/2985)
+* Docs: route rule name by @guicassolato in [3299](https://github.com/kubernetes-sigs/gateway-api/pull/3299)
+
+
+
+## Other specification changes
+
+* SupportedFeatures field in Gateway Class status are now presented as a list of objects by @LiorLieberman in [#3200](https://github.com/kubernetes-sigs/gateway-api/pull/3200)
```

Comment:
This is a breaking change of an experimental API, although we called this out in the PR and multiple community meetings it may be worth highlighting this in a bigger breaking change section